client/src/App.js

```diff
@@ -20,10 +21,62 @@ import { AuthLayout } from '_shared'
 import { useTranslation } from 'react-i18next'
 import { useAuthentication } from '_shared/_hooks/useAuthentication'
 
-const App = () => {
+const Routes = () => {
   const { t } = useTranslation()
   const isAuthenticated = useAuthentication()
+  let location = useLocation()
+
+  return (
+    <div
+      className={`text-primaryBlue font-proxima text-sm h-screen ${
+        location.pathname === '/signup' || location.pathname === '/login'
```

Comment:
wow wow wow 😭 this picture is going to be the death of us; rather than grabbing by location pathname, is there any chance we could do "if the currently-rendered component is wrapped in an AuthLayout, overflow-hidden, else don't do anything"?  Or is that too meta?

Reply:
yes, I remember hearing rumblings re: this picture. Similarly, I had the idea of inserting this `overflow-hidden` class into the top-level div in the AuthLayout component, but unfortunately it wasn't top level enough to have the effect we need. let me see/ think if there is a way to identify AuthContent as the component rendered for the conditionally here, bc this top-level div is the div we need to add the class.